SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialLatencyTracer.cpp

```diff
@@ -231,11 +264,12 @@ USpatialLatencyTracer* USpatialLatencyTracer::GetTracer(UObject* WorldContextObj
 	return nullptr;
 }
 
-bool USpatialLatencyTracer::BeginLatencyTrace_Internal(const AActor* Actor, const FString& FunctionName, const FString& TraceDesc)
+bool USpatialLatencyTracer::BeginLatencyTrace_Internal(const AActor* Actor, const FString& FunctionName, const FString& TraceDesc, FSpatialLatencyPayload& LatencyPayload)
 {
 	FScopeLock Lock(&Mutex);
 
-	TraceKey Key = CreateNewTraceEntry(Actor, FunctionName);
+	bool KeepTrackingReference = GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking();
```

Comment:
Instead of doing different logic inside `CreateNewTraceEntry`, and at the bottom of each of these functions, can we consolidate it to one spot. ie. At the bottom of Begin and Continue functions, if we're not running in Spatial, clean up the maps appropriately?